typescript/infra/scripts/debug-message.ts

```diff
@@ -66,7 +66,8 @@ async function checkMessage(
   multiProvider: MultiProvider<any>,
   message: DispatchedMessage,
 ) {
-  console.log(`Leaf index: ${message.leafIndex.toString()}`);
+  const messageId = utils.messageId(message.message);
+  console.log(`Id: ${messageId}`);
```

Comment:
I think still logging the nonce (i.e. leaf index) could be useful

Reply:
We do this when we log `message.parsed`